'name' is an undefined name twice in evaluation.py

An _undefined name_ may cause NameError to be raised at runtime.

flake8 testing of https://github.com/dmlc/gluon-nlp on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
../gluonnlp/embedding/evaluation.py:380:67: F821 undefined name 'name'
                '{} is not a WordEmbeddingAnalogyFunction'.format(name))
                                                                  ^
./gluonnlp/embedding/evaluation.py:439:67: F821 undefined name 'name'
                '{} is not a WordEmbeddingAnalogyFunction'.format(name))
                                                                  ^
```

--- a/gluonnlp/embedding/evaluation.py
+++ b/gluonnlp/embedding/evaluation.py
@@ -377,7 +377,8 @@ class WordEmbeddingSimilarity(HybridBlock):
 
         if not isinstance(self.similarity, WordEmbeddingSimilarityFunction):
             raise RuntimeError(
-                '{} is not a WordEmbeddingAnalogyFunction'.format(name))
+                '{} is not a WordEmbeddingAnalogyFunction'.format(
+                    self.similarity.__class__.__name__))
 
     def hybrid_forward(self, F, words1, words2, weight):  # pylint: disable=arguments-differ
         """Predict the similarity of words1 and words2.
@@ -436,7 +437,8 @@ class WordEmbeddingAnalogy(Block):
 
         if not isinstance(self.analogy, WordEmbeddingAnalogyFunction):
             raise RuntimeError(
-                '{} is not a WordEmbeddingAnalogyFunction'.format(name))
+                '{} is not a WordEmbeddingAnalogyFunction'.format(
+                    self.analogy.__class__.__name__))
 
     def forward(self, words1, words2, words3):  # pylint: disable=arguments-differ
         """Implement forward computation.

